bundle-workflow/Jenkinsfile

```diff
@@ -6,6 +6,7 @@ pipeline {
     triggers {
       parameterizedCron '''
         H */2 * * * %INPUT_MANIFEST=1.1.0/opensearch-1.1.0.yml
+        H */2 * * * %INPUT_MANIFEST=1.2.0/opensearch-1.2.0.yml
```

Comment:
Not clear to me how the parameters work in Jenkins. Will it 2 different workflows for both params?

Reply:
See https://www.jenkins.io/doc/book/pipeline/syntax/ for Jenkins cron syntax.

Reply:
I understand that, my question is since now we are providing 2 input manifests here, will the job run twice for each manifest?

Reply:
If it's cron syntax then no, each line is its own command, but I could be wrong.

Reply:
Each line is a new run this is how I do it in snapshots creation.